src/definitions/modules/dimmer.less

```diff
@@ -295,14 +295,14 @@ body.dimmable > .dimmer {
     -webkit-transform: translateY(calc(-50% - .5px));
   }
 
-  .ui.segment > .ui[class*="top dimmer"] {
+  .ui.segment > .ui.ui[class*="top dimmer"] {
       border-bottom-left-radius: 0 !important;
```

Comment:
Are these !important still needed?

Reply:
I'll check that and possibly create another PR. I guess dimmer borders are declared in other components as well

Reply:
You were right 👍 Turns out, there was no need for the `!important` settings after checking all other components. To avoid merge conflicts i decided to make their removedment part of this PR, so please review again.
I also updated the fixed jsfiddle testcase  in the description accordingly